packages/warriorjs-abilities/src/directionOf.js

```diff
@@ -2,7 +2,7 @@ import { BACKWARD, FORWARD, LEFT, RIGHT } from '@warriorjs/geography';
 
 function directionOf() {
   return unit => ({
-    description: `Returns the direction (${FORWARD}, ${RIGHT}, ${BACKWARD} or ${LEFT}) to the given space.`,
+    description: `Returns the direction (\`'${FORWARD}'\`, \`'${RIGHT}'\`, \`'${BACKWARD}'\` or \`'${LEFT}'\`) to the given space.`,
     perform(space) {
```

Comment:
This is missing the `params`, and for Senses (which always return something), we could add a `returns`:

```suggestion
    params: [{
      name: 'space',
      type: 'Space',
      description: 'The space.',
    }],
    returns: {
      type: 'string'
      description: 'The direction to the space.',
    },
    perform(space) {
```